service/app/commands/handler_migration_import_data_from_gossb.go

```diff
@@ -59,6 +60,11 @@ func NewImportDataFromGoSSB(
 	}, nil
 }
 
+type ImportDataFromGoSSBResult struct {
```

Comment:
I use this mainly for testing, I haven't committed the program I wrote that needs this yet.